bravado/fido_client.py

```diff
@@ -66,15 +76,22 @@ def headers(self):
         return self._headers
 
     def json(self, **_):
+        # type: (typing.Any) -> typing.Mapping[str, typing.Any]
         # TODO: pass the kwargs downstream
         return self._delegate.json()
 
 
-class FidoClient(HttpClient):
+class FidoClient(typing.Generic[T], HttpClient):
```

Comment:
I don't think this is correct generic typing, as the FidoClient class instance is not dependent on this type T - the FidoFutureAdapter is, which is being wrapped by a HttpFuture. Adding the generic typing to HttpFuture is the right move here.

Reply:
good point
The client itself is not bounded to types (with this I noticed that `bravado.requests_client` is missing typing too)